### PR TITLE
Add rails conf 2023 AI talk

### PR DESCRIPTION
**Pull request by thedayisntgray**
I want to contribute to this list and help the maintainers, so:

- [x] I've accepted the terms of the `CC0` License;
- [x] I've added the topic `rubyml` to the contributed repository if appropriate.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -276,6 +276,9 @@ section on the [Data Science with Ruby][ds-with-ruby] list.
 
 ## Articles, Posts, Talks, and Presentations
 
+- 2020
```
Comment (arbox) [inline readme.md]: I guess it should be 2022?

Comment (thedayisntgray) [reply readme.md]: Good catch! Commit amended

**Comment by thedayisntgray** [comment]
@arbox - Updated the year

**Comment by arbox** [comment]
@thedayisntgray, thank you for your interest to contribute to this list!

**Comment by thedayisntgray** [comment]
> @thedayisntgray, thank you for your interest to contribute to this list!

Thank You @arbox ,

This is such an incredible resource! I love it.

**Review by myronyuktaras — CHANGES_REQUESTED** [review]
**{{The evolution of the brain, the heart, the soul }}
,

**Pull request merged**